packages/base-map/src/index.story.tsx

```diff
@@ -24,7 +24,8 @@ const center: [number, number] = [45.522862, -122.667837];
 export default {
   args: { center },
   component: BaseMap,
-  title: "BaseMap"
+  title: "BaseMap",
+  parameters: { storyshots: { disable: true } }
```

Comment:
Move variable `disableStoryShots` to above this declaration and use it here... Do we need to explicity add it to the other stories? Same thing in the other packages.